app/templates/index.html

```diff
@@ -32,7 +32,7 @@
 			<button onclick="national_button_fun()">National level</button>
 	  </div>
 	  <div id="state" class="cell small-6 level sidebar__btn">
-			<button onclick="">State level</button>
+			<button onclick="state_button_fun()">State level</button>
```

Comment:
Is there a reason why you added it again? On the UX side it does not make sense to click on State level if no state is selected.